### PR TITLE
Add --perf for easily running perf only configuration

### PR DESCRIPTION
**Pull request by paulirish**
`--perf` instead of `--config-path=$PWD/lighthouse-core/config/perf.example-custom.json`

image

### CODE REVIEW

File: lighthouse-cli/index.js [hunk 1]
```diff
@@ -63,6 +66,7 @@ const cli = yargs
     'list-all-audits': 'Prints a list of all available audits and exits',
     'list-trace-categories': 'Prints a list of all required trace categories and exits',
     'config-path': 'The path to the config JSON.',
+    'perf': 'Use a performance-only configuration',
```
Comment (brendankenny) [inline lighthouse-cli/index.js]: maybe something like 'Use a performance-test-only configuration'?

**Comment by samccone** [comment]
I am a bit mixed on this... this feels like flag bloat I would prefer something like

`lighthouse --preset=perf` or something instead

**Comment by paulirish** [comment]
> I am a bit mixed on this... this feels like flag bloat I would prefer something like lighthouse --preset=perf or something instead

As far as smart CLI design practices, I agree that `lighthouse --preset=perf` is superior. But as a lighthouse user, I would still want the this shorthand flag for convenience.

Right now, there are not any presets beyond perf, and making presets into a _Real Thing_ right now means resolving presets w/ custom configs and also w/ custom audits, etc.  Because that turns into a larger conversation and effort, I'd like to defer it, even if it means a possible migration story for `--perf` in the future.

File: lighthouse-cli/index.js (hunk 1 above)
Comment (addyosmani) [inline lighthouse-cli/index.js]: +1 to the suggestion of improving the wording here. This little tweak is nice.

**Comment by addyosmani** [comment]
Gonna side with Paul on this one. I think the right long-term thing to do here would be designing a better way to support multiple presets, but short term as a user I like the simplicity and terseness of `--perf`

**Pull request merged**